### ISSUE TITLE
Language dependency

### ISSUE THREAD

**Issue by gooran** [clovaai/aasist#3]
Hi, This model is language independent. isn't it? I get negative score for Persian bonafide input. I guess that the pre-trained model is dependent to English language. Is there any condition for input audio?

**Comment by Jungjee** [comment]
Hello, did you apply softmax non-linearity to the output and check the value of bona-fide node?
